pages/common/hsw-cli.md

```diff
@@ -0,0 +1,36 @@
+# hsw-cli
+
+> The command line REST tool for the Handshake wallet.
+> More information: <https://npmjs.com/package/hs-client>.
+
+- Unlock the current wallet:
+
+`hsw-cli unlock {{passphrase}} {{timeout}}`
```

Comment:
What is the unit of timeout ? Maybe good to mention that in the description.

Reply:
It's in seconds, I'll add that to the description.